### PR TITLE
Update fingerprint to registered multicodec value.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Addresses issue #4.

### CODE REVIEW

File: lib/X25519KeyPair.js [hunk 1]
```diff
@@ -170,19 +170,57 @@ export class X25519KeyPair extends LDKeyPair {
...
+   * ```
+   *
+   * @param {object} keyNode - A plain object.
+   *
+   * @return {object} The keyNode with an encrypted private key attached.
+   */
+  async addEncryptedPrivateKey(keyNode) {
+    if(this.passphrase !== null) {
+      throw new Error('Encrypted export not yet implemented.');
+    } else {
```
Comment (dlongley) [inline lib/X25519KeyPair.js]: Else not needed

File: lib/X25519KeyPair.js [hunk 2]
```diff
@@ -170,19 +170,57 @@ export class X25519KeyPair extends LDKeyPair {
...
+   *   publicKeyBase58: '8y8Q4AUVpmbm2VrXzqYSXrYcAETrFgX4eGPJoKrMWXNv',
+   *   privateKeyBase58: '95tmYuhqSuJqY77FEg78Zy3LFQ1cENxGv2wMvayk7Lqf'
+   * }
+   * ```
+   *
+   * @param {object} keyNode - A plain object.
+   *
+   * @return {object} The keyNode with an encrypted private key attached.
+   */
+  async addEncryptedPrivateKey(keyNode) {
```
Comment (dlongley) [inline lib/X25519KeyPair.js]: This is unrelated to this PR (I think) ... don't know what it's about, ignoring.

Comment (dmitrizagidulin) [reply lib/X25519KeyPair.js]: Yeah (it's the method that enables `.export()` to work).

File: lib/X25519KeyPair.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/X25519KeyPair.js]: Thanks, fixed.
